docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_Coverflow.htm

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+	<script type="text/javascript" src="../snippet.js"></script>
+	<meta http-equiv="content-type" content="text/html; charset=utf-8">
+	<link href="../styles.css" rel="StyleSheet" type="text/css">
+	<link href="../snippet.css" rel="StyleSheet" type="text/css">
+	<title>Coverflow</title>
+</head>
+<body onload="prettyPrint()" id="content">
+
+
+<h1>Coverflow</h1>
+
+<p>The coverflow component visually applies various effects to selected images.</p>
+
+<img src="../../images/wearable_coverflow.png" width="10%">
+
+<h2>Table of Contents</h2>
+
+<ol class="toc">
+
+		<li><a href="#default-selectors-coverflow-wearable">Default Selectors</a></li>
+
+		<li><a href="#html-examples-coverflow-wearable">HTML Examples</a></li>
+
+		<li><a href="#options-for widget-coverflow-wearable">Options</a></li>
+
+		<li><a href="#methods-coverflow-wearable">Methods</a></li>
+
+</ol>
+
+<h2><a id="default-selectors-coverflow-wearable"></a>Default Selectors</h2>
+
+<p>Elements with the <code>class=&quot;ui-coverflow&quot;</code> attribute can create a Coverflow.<br>
+Changing the value of <code>data-effect=&quot;coverflow&quot;</code> applies a variety of effects.<br>
```

Comment:
It would be great if what kind of effects we have. I guess we have "coverflow", "carousel", "wheel", and "flat". If so, how about adding simple description for each option?

Reply:
ditto.

done.